src/components/Footer/FooterList/FooterList.styled.js

```diff
@@ -1,40 +1,44 @@
 import styled from 'styled-components';
 import { Link } from 'react-router-dom';
 
-const levelChangeSize = '(max-width: 580px)';
+import { device } from '../../../utils';
 
 export const FooterMenuList = styled.ul`
-  @media ${levelChangeSize} {
-    width: 100%;
+  width: 100%;
+
+  @media ${device.laptop} {
+    width: inherit;
   }
 `;
 
 export const FooterMenuItem = styled.li`
   &:not(:last-child) {
-    margin-bottom: 16px;
+    margin-bottom: 0;
 
-    @media ${levelChangeSize} {
+    @media ${device.laptop} {
       &:not(:last-child) {
```

Comment:
you are already in "not(:last-child)" block. Why do you make duplicate this block again?